StoreKitUnitTests/StoreProductTests.swift

```diff
@@ -215,7 +215,7 @@ class StoreProductTests: StoreKitConfigTestCase {
         var priceFormatter = try XCTUnwrap(storeProduct.priceFormatter)
         var productPrice = storeProduct.price as NSNumber
 
-        expect(priceFormatter.string(from: productPrice)) == "4,99 €"
+        expect(priceFormatter.string(from: productPrice)) == "€4.99"
```

Comment:
This is wrong now 😕 that's not how prices are formatted in Euro. I guess the locale is wrong?

Reply:
Ohhh... `formatter.locale = .autoupdatingCurrent` isn't affected by `SKTestSession` but it instead uses the locale set in the test scheme 🤦‍♂️ 

So... going to have mock the locale somehow 🤷‍♂️

Reply:
Well no, the test is correct, this would fail in real code too. I think it's because we need to use the actual locale in the product after all.

Reply:
I'm super confused now 😔 We can only get the locale from SK1 products (which is the SK1 tests were testing for `4,99 €`). The SK2 tests can't get the locale so they using `autoupdatingCurrent` which were returning `€4.99`

But doesn't my setting of the locale in the tests essentially mock the `.autoupdatingCurrent`? In the US I'll get `€4.99` but in Spain I'll get `4,99 €`?

But there is another thing to consider... Only SK1 is using `priceFormatter` for `localizedPriceString`. SK2 is using the `displayPrice` from the SK2 object (which whatever format Apple returns). This may also lead to some inconsistency 😬 

Do we want `localizedPriceString` and `priceFormatter` to return the same for SK1 and SK2 or use the StoreKit default?